src/plugins/kibana_usage_collection/server/collectors/management/schema.ts

```diff
@@ -0,0 +1,116 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { MakeSchemaFrom } from 'src/plugins/usage_collection/server';
+import { UsageStats } from './telemetry_management_collector';
+
+// Retrieved by changing all the current settings in Kibana (we'll need to revisit it in the future).
+// I would suggest we use flattened type for the mappings of this collector.
```

Comment:
Flattened field types won't allow us to do aggregations and answer questions such as: "what is the average precision used in tile maps?" or "what is the average size of terms aggregations users set?"